app/models/rental.rb

```diff
@@ -0,0 +1,34 @@
+class Rental < ApplicationRecord
+  belongs_to :customer
+  belongs_to :video
+
+  def self.inventory_check_out(rental) 
```

Comment:
Style: I would have made these instance methods since you pass an instance in anyway.